### ISSUE TITLE
Multisite config

### ISSUE THREAD

**Issue by andyburnsco** [aghstrategies/com.aghstrategies.airmail#3]
Can this extension work on a multisite where each domain pays for it's own usage at Amazon SES?

If not, how much work is involved to make it domain specific?

**Comment by agh1** [comment]
At least for now, the mail is sent via the standard SMTP system built into CiviCRM.  If each domain can have its own SMTP credentials, you can configure each to use different AWS accounts for SES.  Airmail is most active on bounce notifications, etc. coming inbound.  I have never tested it, but I expect that if each domain's SES is configured with a SNS topic pointing to the endpoint at the corresponding domain, it ought to work fine.

So definitely budget some time for testing and maybe a little for edge case surprises, but there should be no obstacle in principle to doing what you're describing.

**Comment by andyburnsco** [comment]
Excellent, that sounds promising. 

On multisite, we see that the Airmail settings (https://example.org/wp-admin/admin.php?page=CiviCRM&q=civicrm%2Fairmail%2Fsettings&reset=1) are not saving correctly. I tried to save multiple times and the secret code stays blank. I then cleared Civi cache and then it showed my previously inserted code. 

Trying to re-save to something else doesn't work.

I see in civicrm_settings the entry for my 2 sites I tested on. It created 2 endpoints for each site.
